ARCHITECTURE.md

```diff
@@ -0,0 +1,60 @@
+Project Architecture
+===
+
+## Stack and developer requirements
+
+* This project uses typescript and is built with [NPM](https://www.npmjs.com/get-npm)
+
+### Desktop application requirements
+
+* The desktop application is built with [Tauri](https://tauri.studio/). While this project does not have any significant rust code yet, this may be built out in the future. Running the application as a desktop app requires [Rust](https://www.rust-lang.org/) installed.
+
+## Project design
+
+There are two main concerns separated in this project. Data must be collected from users and that data must be rendered into a PDF file when the user exports their 1040 and attachments.
+
+In order to separate these concerns, the data collected from the webforms is stored in a single data object, with schema defined in `src/redux/data.ts`: 
+
+As of now, the crucial root schema is:
```

Comment:
Maybe could change to 

_The root schema of stored data from form submissions is:_

or similar